### DISCUSSION TITLE
Errors upgrading from 2.3.130

### DISCUSSION THREAD

**Discussion by fmtheilig** [Security-Onion-Solutions/securityonion#8918] [Unsupported Versions]
When I run sudo soup on my standalone system, it will fail and I have to restore from backup. I've tried several times hoping a new release would work. In the soup.log file is:

> Running a highstate. This could take several minutes.
> local:
>     Data failed to compile:
> ----------
>     Rendering SLS 'base:filebeat' failed: Jinja variable the template 'filebeat/map.jinja' (imported on line 22) does not export the requested name 'SO'

This happens three times. I captured the output:

> Traceback (most recent call last):
>   File "/usr/lib/python3.6/site-packages/salt/utils/templates.py", line 502, in render_jinja_tmpl
>     output = template.render(**decoded_context)
>   File "/usr/lib/python3.6/site-packages/jinja2/environment.py", line 1090, in render
>     self.environment.handle_exception()
>   File "/usr/lib/python3.6/site-packages/jinja2/environment.py", line 832, in handle_exception
>     reraise(*rewrite_traceback_stack(source=source))
>   File "/usr/lib/python3.6/site-packages/jinja2/_compat.py", line 28, in reraise
>     raise value.with_traceback(tb)
>   File "<template>", line 97, in top-level template code
> jinja2.exceptions.UndefinedError: the template 'filebeat/map.jinja' (imported on line 22) does not export the requested name 'SO'
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "/usr/lib/python3.6/site-packages/salt/utils/templates.py", line 261, in render_tmpl
>     output = render_str(tmplstr, context, tmplpath)
>   File "/usr/lib/python3.6/site-packages/salt/utils/templates.py", line 509, in render_jinja_tmpl
>     raise SaltRenderError("Jinja variable {}{}".format(exc, out), buf=tmplstr)
> salt.exceptions.SaltRenderError: Jinja variable the template 'filebeat/map.jinja' (imported on line 22) does not export the requested name 'SO'
> [CRITICAL] Rendering SLS 'base:filebeat' failed: Jinja variable the template 'filebeat/map.jinja' (imported on line 22) does not export the requested name 'SO'
> [INFO    ] Executing command 'ls' in directory '/root'
> local:
>     Data failed to compile:
> ----------
>     Rendering SLS 'base:filebeat' failed: Jinja variable the template 'filebeat/map.jinja' (imported on line 22) does not export the requested name 'SO'
> 

This was installed from the CentOS ISO about a year ago. It has version 3.6.8 of Python running. I'd hate to do a full rebuild. Any ideas?

**Comment by dougburks** [comment]
Did you ever modify the Filebeat configuration? If so, have you double-checked your modifications to make sure there are no typos?

**Reply by fmtheilig** [reply]
I don't recall modifying any config file other than the global.sls and *_standalone.sls files for quieting and disabling rules. I don't recall making any changes or having any issues prior to the problem appearing, but I don't log changes. Homelab, you understand. I will look at the files referenced scanning for obvious corruption, but I assume I'll have to do a fresh build. Not necessarily a bad exercise under the circumstances.

Thanks.
